src/plugins/credential_management/public/components/create_credential_wizard/create_credential_wizard.tsx

```diff
@@ -176,6 +184,7 @@ export class CreateCredentialWizard extends React.Component<
 
   createCredential = async () => {
     const { savedObjects } = this.context.services;
+    this.setState({ isLoading: true });
     try {
       // TODO: Add rendering spanner https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2050
```

Comment:
Remove todo comment as you have developed the loading mask

Reply:
Thanks for your reminding! I did not notice this todo comment before. Removed